_posts/2024-10-15-anaconda-defaults.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: Ananconda, defaults, and how to not get sued
+
+date: 2024-10-15 00:00:00
```

Comment:
this field also isn't necessary, it's captured from the filename.

Reply:
Oops, my bad!

Yeah, the ordering was what I was trying to do at the time :sweat_smile:  and probably went overboard :grimacing: 

Noted for the future!